frontend/tet/shared/src/components/posting/postingContent/PostingContent.tsx

```diff
@@ -28,6 +36,24 @@ const LocationMap = dynamic(
   }
 );
 
+const keywordList = (list: OptionType[], color: string): JSX.Element => (
+  <>
+    {list.map((keyword: OptionType) => (
+      <li>
+        <Tag
+          theme={{
+            '--tag-background': `var(--color-${color})`,
```

Comment:
Tässä voinee käyttää ennaltamääritettyjä värejä `const theme = useTheme()` ja sitten 
`'--tag-background': theme.colors[color],`
`color` type ei sitten voi olla enää varmaankaan string vaan pitää käyttää jotain `typeof themes.colors` tms määrettä theme-objektista. Se löytyy tuolta: https://github.com/City-of-Helsinki/yjdh/blob/TETP-change-keyword-tags-position-in-posting-view/frontend/shared/src/styles/theme.ts